pkg/bundle/bundle.go

```diff
@@ -201,12 +202,9 @@ func (b *bundle) reconcileBundle(ctx context.Context, req ctrl.Request) (result
 		secretTarget    targetKind = "Secret"
 	)
 
-	type targetResource struct {
-		Kind targetKind
-		types.NamespacedName
-	}
-
-	targetResources := map[targetResource]bool{}
+	targetResources := map[targetKind]map[types.NamespacedName]bool{}
+	targetResources[configMapTarget] = map[types.NamespacedName]bool{}
+	targetResources[secretTarget] = map[types.NamespacedName]bool{}
```

Comment:
Why did you change this?

Reply:
Because I now use nested maps. I think it made a lot of sense for the rest of the code, but that requires initialization of the nested maps.

Reply:
Sorry, that is what I meant: why are we using a nested map?

Reply:
I tried to answer that already: It does not require creating a custom type and I think it makes the loops more compact and easier to read. But I can revert to the current structure.

Reply:
Reverted the attempt to use nested maps.